geonode_mapstore_client/client/js/plugins/save/SaveModal.jsx

```diff
@@ -39,6 +39,11 @@ function SaveModal({
     const [description, setDescription] =  useState('');
     const [nameValidation, setNameValidation] =  useState(false);
 
+    const checkIsEditMode = () => {
```

Comment:
Why do we need to modify this component? Could we directly remove the save modal from the Save plugin?